### ISSUE TITLE
sklern RandomForestRegressor with multiple regression targets dosnt seem to be supported

### ISSUE THREAD

**Issue by speedfreakw** [microsoft/hummingbird#362]
I'm trying to convert an sklern RandomForestRegressor that predicts multiple targets to PyTorch, but it doesn't seem to work or at least Hummingbird does not support this yet, which is not noted in the "supported" section.

If I'm running the following example with only one target (**n_targets = 1** ) everything works as intended:

```
import numpy as np
from sklearn.ensemble import RandomForestRegressor
from sklearn.datasets import make_regression
from hummingbird.ml import convert

X, y = make_regression(n_features=4, n_informative=2, n_targets = 1, random_state=0, shuffle=False)
regr = RandomForestRegressor(max_depth=2, random_state=0)
regr.fit(X, y)
print(regr.predict([[0, 0, 0, 0]]))

# Use Hummingbird to convert the model to PyTorch
model = convert(regr, 'torch')

# Run predictions on CPU
prediction = model.predict([[0.0, 0.0, 0.0, 0.0]])
print(prediction)
```

but if I change the Targets to **n_targets = 2**, I get the following error Message at `model = convert(regr, 'torch')`:

> Unable to find converter for SklearnRandomForestRegressor type <class 'NoneType'> with extra config: {'container': True, 'n_threads': 6, 'n_trees': 100}.

**Comment by interesaaat** [comment]
Hi @speedfreakw, thanks for finding this. Yes, we currently don't support multiple regression targets. Do you want to give it a shot? 

Another thing that I noticed is that the error returned is not that helpful, we should improve on that too.

**Comment by speedfreakw** [comment]
Hi @interesaaat, thanks for the information!
Unfortunately I don't have the time to give it a shot myself at the moment, as I am working on my master thesis right now.

**Comment by interesaaat** [comment]
How badly do you need this feature? I can try to see if I can fix it quickly (although I don't think it will be an easy fix).

**Comment by speedfreakw** [comment]
I have not yet found an alternative. So it would be really awsome if you could see if you could fix it quickly. My problem is that the sklern-model is very bad at making a single prediction quickly (takes about 230ms) and I want to use it for realtime processing of depth images. So I need to cut the processing time below 30ms somehow.

**Comment by speedfreakw** [comment]
I could also try to train 6 separate RandomForests (I need 6 targets), which I could convert to PyTorch and see if the total processing time of these 6 is less than if I use a single sklern random forest

**Comment by interesaaat** [comment]
You will have to call the models in parallel somehow. If you have GPU, also doing inference sequentially should be ok.

**Comment by speedfreakw** [comment]
I will try that tomorrow. 
I also just noticed that there appears to be a bug if I try to assign the model to the GPU. I will open an seperate Issue with on that later.

```
# Run predictions on GPU
model.to('cuda')
model.predict(X)
```

> 'PyTorchSklearnContainerRegression' object has no attribute 'to'

**Comment by interesaaat** [comment]
Yea this should work. I think the `to` function got removed inadvertently in some PR.

**Comment by interesaaat** [comment]
Closed in #430.
